python/persistenceClustering1.py

```diff
@@ -0,0 +1,124 @@
+from paraview.simple import *
+
+# create a new 'CSV Reader'
+clusteringcsv = CSVReader(FileName=["clustering1.csv"])
+
+# create a new 'Table To Points'
+tableToPoints1 = TableToPoints(Input=clusteringcsv)
+tableToPoints1.XColumn = "X"
+tableToPoints1.YColumn = "Y"
+tableToPoints1.ZColumn = "X"
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.